src/index.module.css.json

```diff
@@ -0,0 +1 @@
+{"main":"_main_ubzcz_1","track":"_track_ubzcz_5","wrapper":"_wrapper_ubzcz_9","inner":"_inner_ubzcz_15"}
```

Comment:
not sure if it matters, but this has a lint error (missing empty EOL)

Reply:
Done in eeb6fee.